src/main/java/fr/moribus/imageonmap/ImageOnMap.java

```diff
@@ -69,6 +63,11 @@ public File getImageFile(short mapID)
         return new File(imagesDirectory, "map"+mapID+".png");
     }
     
+    public File getFullImageFile(short mapIDstart, short mapIDend)
```

Comment:
This should go into the `image.ImageMap` class (and without parameters, using its attributes).